pkg/member/member.go

```diff
@@ -488,17 +484,9 @@ func (m *EmbeddedEtcdMember) SetMemberDeployPath(id uint64) error {
 	return nil
 }
 
-func (m *EmbeddedEtcdMember) getMemberGitHashPath(id uint64) string {
```

Comment:
Do we need to add a test to check if the changes produce the same results as before? For example, we could output the path from the old version as a dataset for the new code test.

Reply:
All I can think of is to verify the consistency of the paths through manual testing. There doesn't seem to be any good testing methods to automate🤔

Reply:
Maybe we can use fixed testdata from your manual testing.